pkg/linstorhelper/properties.go

```diff
@@ -46,6 +46,10 @@ func MakePropertiesModification(current, expected map[string]string) *lclient.Ge
 		}
 	}
 
+	if len(lastApplied) > 0 && len(expected) == 0 {
```

Comment:
Why only when `len(lastApplied) > 0`? If we're deleting all the properties we have set, then we can delete `LastApplyProperty` immediately as well, right?

Reply:
True, don't really know why this check existed.